public/css/style.css

```diff
@@ -4,6 +4,7 @@
   /* border: 1px solid red; */
 }
 
+/* layout.ejs */
 body {
```

Comment:
Devem ser usadas classes e não a tag body

Reply:
Camila, aqui não tem problema utilizar a tag, por que imagino que seja algo para ser herdado pra todos componentes dentro do body. Mas caso não for, tu está correta.